modules/python/python-logmsg.h

```diff
@@ -32,6 +32,7 @@ typedef struct _PyLogMessage
   PyObject_HEAD
   LogMessage *msg;
   PyObject *bookmark_data;
+  gboolean config_version_3_x_compat;
```

Comment:
I would prefer this to be called something that is less focusing on version numbers. In ValuePairs, this was called "cast_to_strings"

Reply:
Thanks, fixed.